lib/config.ts

```diff
@@ -150,6 +150,36 @@ export function get(name: string): any
                 default: 50,
                 arg: 'throttle-rate'
             }
+        },
+        'webSocket': {
```

Comment:
For options, especially for command-line arguments, let's go with `websocket`.  I know `web` and `socket` are distinct words, but this is also for configuration and aren't code variables.